File: Backend 1 - Intro a Java/Guia 18 - JavaScript/Ejercicios/ej_4/ejercicio_4.js
// Realiza un programa que sólo permita introducir los caracteres ‘S’ y ‘N’. Si el usuario
// ingresa alguno de esos dos caracteres se deberá de imprimir un mensaje por pantalla
// que diga “CORRECTO”, en caso contrario, se deberá imprimir “INCORRECTO”.

var caracter = ((prompt("Ingrese 'S' o 'N'")).toString()).toUpperCase();

var estado = (caracter == 'S' || caracter == 'N') ? true : false;

(estado) ? alert("CORRECTO") : alert("INCORRECTO");

File: Tramo 2 - Backend 1/Guia 03 - Estruct. de control/Ejercicios/Ejercicio4.java
/// EJERCICIO 4
// Escriba un programa que pida una frase o palabra y valide si la primera letra
//  de esa frase es una ‘A’. Si la primera letra es una ‘A’, se deberá de imprimir
//   un mensaje por pantalla que diga “CORRECTO”, en caso contrario, se deberá imprimir “INCORRECTO”.

import java.util.Scanner;

public class Ejercicio4 {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String palabra;

        System.out.println("Ingrese una palabra:");
        palabra = in.next();

        if((palabra.substring(0, 1)).equalsIgnoreCase("a")){
            System.out.println("CORRECTO");
        }else{
            System.out.println("INCORRECTO");
        }
        in.close();
    }
}
